README.md

```diff
@@ -40,14 +40,23 @@ https://www.codementor.io/mercurial/how-to-install-node-js-on-macos-sierra-mphz4
 nvm install 8.10.0
 ```
 
-#### 4. run the backend application
+#### 4. set backend database credentials
```

Comment:
Can we add the url as a env var as well? I would like to test locally with my MySql instance

Reply:
I can... there are several other differences to the config for the mongo db.. how do you want to manage all of that?